sites/www.seafloorexplorer.org.conf

```diff
@@ -1,12 +1,37 @@
-# Projects where most of the subject URLs are hard-coded to S3.
-# We need to redirect the few that use the project domain instead.
 server {
     include /etc/nginx/ssl.default.conf;
     server_name www.seafloorexplorer.org;
 
-    location /subjects/ {
-        return 301 https://s3.amazonaws.com/$host$request_uri;
-    }
+    location / {
+        resolver 8.8.8.8;
+
+        # Include all proxy headers so that the Host can be changed
+        add_header 'Access-Control-Allow-Origin' '*';
+        add_header 'Access-Control-Allow-Credentials' 'true';
+        add_header 'Access-Control-Allow-Methods' 'GET';
+        add_header 'Access-Control-Allow-Headers' 'DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type';
+        add_header 'X-Content-Type-Options' 'nosniff';
+        add_header 'Content-Security-Policy' "frame-ancestors 'self' $csp_whitelist";
+        add_header 'X-Content-Security-Policy' "frame-ancestors 'self' $csp_whitelist";
+        add_header 'X-XSS-Protection' '1; mode=block';
+
+        proxy_set_header       Host zooniversestatic.z13.web.core.windows.net;
 
-    include /etc/nginx/proxy.conf;
+        proxy_redirect         /$host/ /;
+        proxy_cache            STATIC;
+        proxy_cache_valid      200  1d;
+        proxy_cache_use_stale  error timeout invalid_header updating
+                    http_500 http_502 http_503 http_504;
+        proxy_hide_header Access-Control-Allow-Origin;
+        proxy_hide_header Access-Control-Allow-Credentials;
+        proxy_hide_header Access-Control-Allow-Methods;
+        proxy_hide_header Access-Control-Allow-Headers;
+
+        # Contents of proxy.conf, updated to AZ
+        rewrite (?i)\.(jp(e)?g|gif|png|ico|txt|woff|ttf|mp(3|4)|webm|og(a|g|m|v|x)|spx|opus|pdf|tar|gz|tgz|bz2|tbz2|zip)$ https://static.zooniverse.org/www.seafloorexplorer.org$request_uri;
```

Comment:
this will redirect the client away from this server block and serve the content via the default server block, i believe in here so from s3, https://github.com/zooniverse/static/blob/7f32a9fa8424bca7396198c1e5c9e63d44a1cd40/nginx-proxy.conf#L7

Reply:
Confirmed, this needs to point to Azure instead..